### PR TITLE
add nf-core-project

### PR DESCRIPTION
**Pull request by markjschreiber**
Project with 3 workflows from nf-core

### CODE REVIEW

File: examples/nf-core-project/agc-project.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    readOnly: true
+  - location: s3://encode-public
+    readOnly: true
+contexts:
+  nfCtx:                                  # By default, limited to 4xlarge instance types
+    engines:
+      - type: nextflow
+        engine: nextflow
+  bigMemCtx:
+    instanceTypes: [                      # Standard types plus r5.8xlarge due to memory requirements of some workflows like sarek
```
Comment (AndreyDovydenko) [inline examples/nf-core-project/agc-project.yaml:36]: Let's please stick with Yaml notation for the list too

File: examples/nf-core-project/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## Sarek Workflow
+
+Due to the memory requirements of some steps of the sarek workflow you will need to run this on a deployed `bigMemCtx`
```
Comment (AndreyDovydenko) [inline examples/nf-core-project/README.md:3]: Do we need the small context then?

**Review by tneely — APPROVED** [review]

**Review by wleepang — COMMENTED** [review]
Can you add a README in the project root that describes what these workflows are and provides appropriate link out to the NF-Core and the repos for each of the pipelines?

**Comment by markjschreiber** [comment]
> Can you add a README in the project root that describes what these workflows are and provides appropriate link out to the NF-Core and the repos for each of the pipelines?

Done

**Review by wleepang — APPROVED** [review]

**Pull request merged**